File: js/q16..js
var arr=[3,5,9,1,7]
var count=0

for(i=0;i<arr.length;i++){
    if(arr[i]<0){
        count++
    }
}

if(count==0){
    console.log("Array contain positive numbers.")
}else{
    console.log("Array not contain positive numbers.")
}

File: clangua/q16.c
#include <stdio.h>
#include <string.h>

int main() {
    
    int arr[]={3,5,8,1,7};
    int len=sizeof(arr)/sizeof(arr[0]);
    int count=0,i;
    
    for(i=0;i<len;i++){
        if(arr[i]<0){
            count++;
        }
    }
    
    if(count==0){
        printf("Array contains only positive numbers.");
    }else{
        printf("Array not contains positive numbers");
    }
    
    return 0;
}